packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -361,6 +376,34 @@ $btn-sizeXSmall-lineHeight      : 1;
     line-height: $btn-sizeXSmall-lineHeight;
 }
 
+/**
+* ==========================================================================
+* Icon Size Modifiers
+* ==========================================================================
+*/
+
+.o-icon {
+    background-color: $icon-bgColor;
+    border-radius: 50%;
+    width: $icon-default-width;
+    vertical-align: middle;
+    padding: $icon-default-padding;
+
+    &:hover {
+        background-color: $icon-bgColor--hover;
+    }
+    &:active {
+        background-color: $icon-bgColor--active;
+    }
+}
+
+.o-icon--sizeSmall {
+    width: $icon-sizeSmall-width;
+}
+
+.o-icon-sizeLarge {
+    width: $icon-sizeLarge-width;
+}
```

Comment:
If you chain the modifiers then you can just do this:

```suggestion

     &.c-btn--sizeSmall {
        width: $icon-sizeSmall-width;
    }

    &.o-btn-sizeLarge {
        width: $icon-sizeLarge-width;
    }
}
```